cdc/owner.go

```diff
@@ -85,6 +85,15 @@ func (o *Owner) getMinGCSafePointCache(ctx context.Context) model.Ts {
 			return o.minGCSafePointCache.ts
 		}
 		o.minGCSafePointCache.ts = oracle.ComposeTS(physicalTs-(o.gcTTL*1000), logicalTs)
+		// get minimum safepoint across all services
+		actualGCSafePoint, err := o.pdClient.UpdateServiceGCSafePoint(ctx, CDCMinGCSafePointCacheServiceID, CDCMinGCSafePointCacheServiceTTL, o.minGCSafePointCache.ts)
```

Comment:
- It is not a good semantic to call an `updateXX` API in a `getXX` interface
- Rather than to call `UpdateServiceGCSafePoint` to get min GC safepoint, I would rather to query the gc_safe_point from PD, but seems PD doesn't expose this API directly

Reply:
Do you mean to add an interface in `cdc/kv/etcd.go` that can get the `gc_safe_point` directly from etcd?

Reply:
Yep, is using `gc_safe_point` all right here?

Reply:
I changed the way to get `actualGCSafePoint`.  Please review it again.